src/components/UrlUtilsService.js

```diff
@@ -104,9 +104,9 @@ goog.provide('ga_urlutils_service');
         // Remove proxy from the URL
         this.unProxifyUrl = function(url) {
           if (this.isValid(url)) {
-            var reg = new RegExp(['^(http|https):\/\/(service-proxy.',
-                '(dev|int|prod).bgdi.ch|proxy.geo.admin.ch)',
-                '\/(http|https)\/(.*)'].join(''));
+            var reg = new RegExp(['^(http|https)://(service-proxy.',
```

Comment:
When you're a re using the Regexp constructor you don't need to escape things. I was concerned too but tests passes.